api_v3/services/BaseEntryService.php

```diff
@@ -107,6 +107,11 @@ function addContentAction($entryId, KalturaResource $resource)
 		if ($dbEntry->getStatus() != entryStatus::NO_CONTENT)
 			throw new KalturaAPIException(KalturaErrors::ENTRY_ALREADY_WITH_CONTENT);
 		
+		if($resource->localFilePath && myUploadUtils::handleRestrictedFilesByPath($resource->localFilePath))
```

Comment:
better call to isFileTypeRestricted directly as it returns a boolean and already we throw here an api exception